tests/test_cli.py

```diff
@@ -101,11 +100,12 @@ def test_cli_early_exit(args):
         ),
     ],
 )
-def test_cli_path_mapping(
-    mock_load_json, mock_path, tmp_path, args, raises, expected
-):  # pylint: disable=unused-argument
+def test_cli_path_mapping(mock_load_json, mock_path, tmp_path, args, raises, expected):  # pylint: disable=unused-argument
     with raises:
-        mock_path.return_value = f"{Path(__file__).resolve().parents[0]}" "/test_data/"
+        mock_path.return_value = (
+            f"{Path(__file__).resolve().parents[0]}",
+            "/test_data/",
+        )
```

Comment:
This is changing the return type to a tuple instead of a string. Here and below.

Reply:
Is it interesting / expected that this change didn't break the test?

Reply:
OH that's the source of the `site_packages_dir = "".join(sysconfig.get_path("purelib"))` issue, i'll fix that here and update there.